frontend/src/FileManager/Actions/CreateFolder/CreateFolder.action.jsx

```diff
@@ -38,6 +38,12 @@ const CreateFolderAction = ({ filesViewRef, file, onCreateFolder, triggerAction
       return;
     }
 
+    if (e.key === 'Escape') {
+      e.preventDefault();
+      triggerAction.close();
+      onRefresh();
```

Comment:
@vijay-1842 Thank you for your contribution. Adding a way to cancel folder creation with `ESC` key is a great addition. However, I don't recommend using `onRefresh` here, as this will cause an unnecessary API call.

Instead, we can filter that folder out:
```js
if (e.key === 'Escape') {
      e.preventDefault();
      triggerAction.close();
      setCurrentPathFiles((prev) => prev.filter((f) => f.key !== file.key));
}
```

Test with it and create PR for merge. Thanks.